### ISSUE TITLE
508-defect-2 [COGNITION]: Related content after the call-to-action button SHOULD be associated using aria-describedby

### ISSUE THREAD

**Issue by jenstrickland** [department-of-veterans-affairs/va.gov-team#16148]
# [508-defect-2](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/accessibility/guidance/defect-severity-rubric.md#508-defect-2)

**Feedback framework**

- **❗️ Must** for if the feedback must be applied 
- **⚠️ Should** if the feedback is best practice 
- **✔️ Consider** for suggestions/enhancements 

**Definition of done**

1. Review and acknowledge feedback.
1. Fix and/or document decisions made.
1. Accessibility specialist will close ticket after reviewing documented decisions / validating fix.

<hr/>

## Point of Contact

**VFS Point of Contact:** _Jennifer_

## User Story or Problem Statement

As a screen reader user, I need to know there is information relevant to the "Request a Higher-Level Review online" button that follows the button, to prevent me from simply selecting the button and not having all the context. 

## Details

While this pattern exists across our introduction pages, the accessibility specialists are documenting this issue. For pages with a call-to-action (CTA) at the top (and sometimes another at the bottom) the associated relevant information that follows may not be apparent for non-sighted users. Adding an `aria-describedby` attribute to the CTA button will let screen reader users know there is additional context that may be useful.

## Acceptance Criteria

- [ ] The "Request a Higher-Level Review online" button has an  `aria-describedby` attribute 

### Current

```html
<div class="vads-u-background-color--gray-lightest vads-u-padding--2 vads-u-margin-top--2">
!  You can request a Higher-Level Review online using <strong>VA Form 20-0996</strong>.

!  <p> 
    <button class="usa-button-primary va-button-primary">Request a Higher-Level Review online</button>
  </p>

!  <a href="/decision-reviews/higher-level-review">
    Learn about other ways you can request a Higher-Level Review
  </a>
</div>
```

### Recommendation

```diff
<div class="vads-u-background-color--gray-lightest vads-u-padding--2 vads-u-margin-top--2">
<!-- if possible, may this text be wrapped in a p tag -->
+ <p>
    You can request a Higher-Level Review online using <strong>VA Form 20-0996</strong>.
  </p>

  <!-- if possible, may the button exist without the p tag? -->
+  <button aria-describedby="unique_id" class="usa-button-primary va-button-primary">
    Request a Higher-Level Review online
  </button>

<!-- if possible, may this link be wrapped in a p tag -->
+ <p id="unique_id">
     <a href="/decision-reviews/higher-level-review">
      Learn about other ways you can request a Higher-Level Review
    </a>
  </p>
</div>
```

## WCAG Guidance
- [ARIA1: Using the aria-describedby property to provide a descriptive label for user interface controls](https://www.w3.org/TR/WCAG20-TECHS/ARIA1.html)

## Screenshots or Trace Logs

![Screen Shot 2020-11-13 at 9 59 00 AM](https://user-images.githubusercontent.com/57469/99086356-659cdb00-2597-11eb-8d20-90ababc07c2b.png)

**Comment by joshkimux** [comment]
I've tested and confirmed that this has been fixed 👍

**Comment by Mottie** [comment]
@joshkimux This fix has been merged in https://github.com/department-of-veterans-affairs/vets-website/pull/15395; tagging you to get it on your radar.

**Comment by Mottie** [comment]
LOL oops - forgot to refresh 😛
